observability/network_observability/network-observability-overview.adoc

```diff
@@ -22,13 +22,11 @@ The Network Observability Operator provides the Flow Collector API custom resour
 [id="no-console-integration"]
 == {product-title} console integration
 
-{product-title} console integration offers overview, topology view, and traffic flow tables in both *Administrator* and *Developer* perspectives.
-
-In the *Administrator* perspective, you can find the Network Observability *Overview*, *Traffic flows*, and *Topology* views by clicking *Observe* -> *Network Traffic*. In the *Developer* perspective, you can view this information by clicking *Observe*. The Network Observability metrics dashboards in *Observe* -> *Dashboards* are only available to administrators. 
+{product-title} console integration offers overview, topology view, and traffic flow tables. The Network Observability metrics dashboards in *Observe* -> *Dashboards* are only available to users with administrator access.
```

Comment:
I think we need articles here. I am certain that the _only_ should be moved closer to the word it refers to, _users_. 
```suggestion
{product-title} console integration offers an overview, a topology view, and traffic flow tables. The Network Observability metrics dashboards in *Observe* -> *Dashboards* are available only to users with administrator access.
```